File: 1672-richest-customer-wealth/1672-richest-customer-wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
  var richestCustomer = 0;
  for (var i = 0; i < accounts.length; i++) {
    var customerBalance = 0;
    for (var j = 0; j < accounts[i].length; j++) {
      customerBalance += accounts[i][j];
    }
        if (customerBalance >= richestCustomer) {
        richestCustomer = customerBalance
        }
    }
    return richestCustomer;
  }

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.py
class Solution(object):
    def maximumWealth(self, accounts):
        """
        :type accounts: List[List[int]]
        :rtype: int
        """
        richest = 0
        for i in accounts:
            for j in accounts:
                if sum(j) > richest:
                    richest = sum(j)
        return richest